qemu/tests/cfg/commit_with_bitmaps_nospace.cfg

```diff
@@ -0,0 +1,28 @@
+- commit_with_bitmaps_nospace:
+    type = commit_with_bitmaps_nospace
+    start_vm = no
+    virt_test_type = qemu
+    not_preprocess = yes
+    vg_name = vg_test
+    lv_name = base
+    lv_size = 128M
+    block_raw_name = test.img
+    emulated_image = images/${block_raw_name}
+    emulated_image_size = 128M
+    remove_emulated_image = yes
+    images = base
+    image_name_base = /dev/${vg_name}/${lv_name}
```

Comment:
let's set image_format_base = qcow2, because only qcow2 image can be added bitmaps

Reply:
@zhencliu , yes, you're right, have set its format in cfg.